app/src/layout/SkillTreeFolderModal.tsx

```
@@ -0,0 +1,429 @@
+              <Lock className="w-3 h-3 mr-1" />
+              Owned (Inactive)
+            </Badge>
+          )}
+        </div>
+      </div>
+
+      {/* Description */}
+      <div
+        className="text-sm text-muted-foreground prose prose-sm dark:prose-invert max-w-none"
+        // eslint-disable-next-line @eslint-react/dom/no-dangerously-set-innerhtml
+        dangerouslySetInnerHTML={{ __html: skill.description }}
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)